backend/app/api/src/helpers/TagHelper.ts

```diff
@@ -0,0 +1,28 @@
+import { Organization } from '@stamhoofd/models';
+import { QueueHandler } from '@stamhoofd/queues';
+import { OrganizationTag, TagHelper as SharedTagHelper } from '@stamhoofd/structures';
+
+export class TagHelper extends SharedTagHelper {
+    static updateOrganizations(oldTags: OrganizationTag[], newTags: OrganizationTag[]) {
+        const queueId = 'update-tags-on-organizations';
+        QueueHandler.cancel(queueId);
+        QueueHandler.schedule(queueId, async function (this: undefined) {
+            const organizations = await Organization.all();
```

Comment:
Organizations moeten langzaam gelooped worden, niet allemaal in één keer in memory geladen worden